lib/CleanPlugin.js

```diff
@@ -443,3 +495,6 @@ class CleanPlugin {
 }
 
 module.exports = CleanPlugin;
+module.exports.getDirectories = getDirectories;
+module.exports.hasFile = hasFile;
+module.exports.isEqualPath = isEqualPath;
```

Comment:
Let's implement them as a plugin methods (using `_` before method, i.e. `_isEqualPath` and for other too), don't want to make them as a part of public API

Reply:
Thanks for your review